pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -328,6 +364,32 @@ func mockLibvirtClient(mockController *gomock.Controller, clientResult libvirtCl
 	return mockClient
 }
 
+func expectUpdateDeviceNotCalled(mockController *gomock.Controller) *cli.MockVirDomain {
+
+	mockClient := cli.NewMockVirDomain(mockController)
+	mockClient.EXPECT().UpdateDeviceFlags(gomock.Any(), gomock.Any()).Times(0).Return(nil)
+
+	return mockClient
+}
```

Comment:
I suspect that this function is redundant, because the mock is supposed to panic in case `UpdateDeviceFlags` is called without declaring that it will be called.
We can handle it in a follow-up.